epios/post_process.py

```diff
@@ -0,0 +1,1423 @@
+
+                    Default = 17
+
+                    *The last group includes age >= some threshold*
+
+                    *This is used when autotuning is turned off*
+                age_group_width : int
+                    Indicating the width of each age group(except for the last group)
+
+                    Default = 5
+
+                    *This is used when autotuning is turned off*
+                sampling_percentage : float, between 0 and 1
+                    The proportion of additional samples taken from a specific (age-)regional group
+
+                    Default = 0.1 (Only for non-responders)
+
+                    *This is used when autotuning is turned off*
+                proportion : float, between 0 and 1
+                    The proportion of total groups to be sampled additionally
+
+                    Default = 0.01 (Only for non-responders)
+
+                    *This is used when autotuning is turned off*
+                threshold : NoneType or Int
+                    The lowest number of groups to be sampled additionally
+
+                    Default = None (Only for non-responders)
+
+                    *This is used when autotuning is turned off*
+                num_age_group_range : list
+                    All possible number of age groups that you want to try/iterate over
+
+                    Default = [10, 13, 15, 17, 20]
+
+                    *The last group includes age >= some threshold*
+
+                    *This is used when autotuning is turned on*
+                age_group_width_range : list
+                    All possible age group width (except for the last group)
+                    that you want to try/iterate over
+
+                    Default = [5, 10]
+
+                    *This is used when autotuning is turned on*
+                sampling_percentage_range : list
+                    All possible proportion of additional samples taken
+                    from a specific (age-)regional group that you want to
+                    try/iterate over
+
+                    Default = [0.1, 0.2, 0.3] (Only for non-responders)
+
+                    *This is used when autotuning is turned on*
+                proportion_range : list
+                    All possible proportion of total groups to be sampled additionally
+                    that you want to try/iterate over
+
+                    Default = [0.01, 0.05, 0.1] (Only for non-responders)
+
+                    *This is used when autotuning is turned on*
+                threshold_range : list
+                    All possible lowest number of groups to be sampled additionally
+                    that you want to try/iterate over
+
+                    Default = [10, 20, 30] (Only for non-responders)
+
+                    *This is used when autotuning is turned on*
+        '''
+        # Check whether metric is recognisable
+        recognisable_metric = [
```

Comment:
Sorry I commented on this earlier! I personally prefer to include these checks in the base level function where the argument is used, but happy if you want to leave this check here